src/service.js

```diff
@@ -1,26 +1,49 @@
 import Debug from 'debug';
+import isArray from "lodash/isArray"
+const ACT_SEND = 'send'
+const ACT_BATCH = 'batch'
 
 const debug = Debug('feathers-sendwithus:service');
 
-export default function createService({ api, templateMapper }) {
+const _execApiCall=function(data,api,callType){
+    api[callType](data, (err, result) => {
+        if (err) {
+            return this.reject(err);
+        }
+        this.resolve(result);
+    });
+}
+
+export default function createService({ api, templateMapper,batchOpts }) {
+  const {path='/api/v1/send',method='POST',size=10}=batchOpts || {}
```

Comment:
`size` is used to determine the chunking size, perhaps it should be named something more descriptive such as `chunkSize`.